Openshift/s2i-images/build-all.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Is app-builder not needed in this script or is it included in one of the child images?

Reply:
The new app-builder image now falls under 'vanilla' docker images. There is a different script to pull them. I can create another script at a higher level to pull all of the images in one go.